RemoteHealthcare/Bluetooth.cs

```diff
@@ -22,62 +19,24 @@ public static async Task<int> SetConnectionAsync(BLE ble, string device, string
             return errorCode;
         }
 
-        private static void BleHeart_SubscriptionValueChanged(object sender, BLESubscriptionValueChangedEventArgs e)
+        private static void BleHeart_SubscriptionValueChanged(object sender, avansBikeData e)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)